### ISSUE TITLE
AMC pruning results in poor acc after 1st episode.

### ISSUE THREAD

**Issue by zeyu-hello** [NervanaSystems/distiller#526]
I am sure I have loaded pretrained resnet50 weights from torchvision to do pruning.
the original acc: `top1: 83.1091, top5: 95.0389, loss: 0.6449`

My training scripts:
```shell
python amc.py --arch=resnet50 --pretrained \
             ImageNet/ \
              --lr=0.05 -b=96 \
              --verbose \
              --amc-protocol=accuracy-guaranteed \
              --amc-action-range 0.0 0.8 \
              --amc-target-density=0.5 \
              -p=50 \
              --amc-prune-pattern=channels \
              --amc-prune-method=fm-reconstruction \
              --amc-agent-algo=DDPG \
              --amc-cfg=./auto_compression_channels.yaml \
              --amc-rllib=hanlab \
              -j=1
```
and results:

```shell
2020-07-20 20:04:57,847 - Best reward=-13.626707058027034  episode=0  top1=36.65037934371975
2020-07-20 20:04:57,852 - Top1: 36.65 - compute: 53.73% - params:52.83% - actions: [0.796875, 0.3125, 0.40625, 0.328125, 0.734375, 0.109375, 0.265625, 0.6875, 0.34375, 0.140625, 0.484375, 0.2265625, 0.40625, 0.375, 0.26953125, 0.359375, 0.0625, 0.796875, 0.63671875, 0.328125, 0.3125, 0.15625, 0.01171875, 0.796875, 0.3203125, 0.35546875, 0.0390625, 0.572265625, 0.248046875, 0.771484375, 0.630859375, 0.40625]
2020-07-20 20:04:57,863 - Resetting the environment (init_only=False)
2020-07-20 20:04:57,891 - +--------------------------------------------------+
2020-07-20 20:04:57,891 - Episode 1 is starting
2020-07-20 20:04:59,656 - Created, applied and saved a thinning recipe
2020-07-20 20:04:59,656 - Episode 1 is ending
```
